LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 Redpanda
```

Comment:
is that year right?

Reply:
That’s when the first commit was on April 20th 2020 (yes, really, by Alex to boot) but I have no idea. My assumption is that literally anything is better than leaving it boilerplate